finality-aleph/src/network/gossip/service.rs

```diff
@@ -200,11 +193,18 @@ impl<N: RawNetwork, AD: Data, BSD: Data> Service<N, AD, BSD> {
     fn send_to_authentication_peer(&mut self, data: AD, peer: N::PeerId) -> Result<(), SendError> {
         match self.get_authentication_sender(&peer) {
             Some(sender) => {
-                match sender.unbounded_send(data) {
+                match sender.try_send(data) {
                     Err(e) => {
                         // Receiver can also be dropped when thread cannot send to peer. In case receiver is dropped this entry will be removed by Event::NotificationStreamClosed
                         // No need to remove the entry here
-                        if e.is_closed() {
+                        if e.is_full() {
+                            warn!(
+                                target: "aleph-network",
+                                "Failed sending data in mpsc_notification_stream_authentication to peer because peer_sender receiver is full: {:?}",
```

Comment:
Do we wanna use these names `mpsc_notification_stream_authentication` if actually nothing's named like this anymore? We could have: "Failed sending data through authentication notification channel ..." or sth like that.

Reply:
Yeah, no point keeping the weird name, the alternative you suggest is better.

Reply:
Agreed & renamed